### DISCUSSION TITLE
Homepage UI Update

### DISCUSSION THREAD

**Discussion by audaro** [scribble-rs/scribble.rs#289] [Show and tell]
I made some changes to the homepages UI and how it responds to mobile usage. Still a WIP.

https://github.com/sayhijordy/scribble.rs

**Comment by Bios-Marcel** [comment]
I'll check it out tomorrow :)

**Reply by Bios-Marcel** [reply]
So, I think I'd even omit the margins around the content on the mobile variant or reduce it heavily.
The lobby list could also save space by being having two columns in the second row.

I am unsure about the color choices though, I generally dislike how it is now with recognizing what's selected.
